flyway/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM flyway/flyway
+WORKDIR /flyway/sql
+COPY ./db/migrations .
```

Comment:
@miraclebg In order to run the migrations we need to place them in specific folder of the image `/flyway/sql`

You might wanna check the ref https://github.com/flyway/flyway-docker#supported-volumes